task8/TutterApp/src/main/java/servlets/CheckServlet.java

```diff
@@ -0,0 +1,15 @@
+package servlets;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.io.IOException;
+
+@WebServlet("/check")
+public class CheckServlet extends HttpServlet {
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws IOException {
+        resp.getWriter().write("{\n\t\"sucess\" : true\n}");
```

Comment:
You don't actually need tabs and newlines here) Json parser in your browser won't grant you extra points for that))
And you can also engage `ObjectMapper` or for instance `org.json.JSONWriter` to  do the job in style ))